### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sandersky**
### This project uses [semver](semver.org), please check the scope of this pr:
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change
# CHANGELOG
- **Fixed** code so it works with versions of Ember all the back to `2.3`.
- **Upgraded** dependencies.

### CODE REVIEW

File: blueprints/ember-frost-modal/index.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ module.exports = {
...
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-elsewhere', target: '~0.3.0'},
+        {name: 'ember-elsewhere', target: '0.4.1'},
         {name: 'ember-frost-core', target: '>=0.23.0 <1.0.0'},
-        {name: 'ember-get-config', target: '~0.1.3'},
+        {name: 'ember-get-config', target: '0.1.11'},
         {name: 'ember-ignore-children-helper', target: '^1.0.0'},
-        {name: 'liquid-fire', target: '~0.24.1'}
+        {name: 'liquid-fire', target: '0.25.0'}
```
Comment (sglanzer-fire) [inline blueprints/ember-frost-modal/index.js]: ~ for all of these

File: bower.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-frost-modal",
   "dependencies": {
-    "ember": "~2.6.0",
+    "ember": "2.8.2",
```
Comment (sglanzer-fire) [inline bower.json]: ~

**Review by sglanzer-fire — CHANGES_REQUESTED** [review]
Can you pin to minor for most of these instead of an explicit patch version?  We're going to hit issues if the addons are too restrictive in semver ranges (this is likely a broad issue for us right now)

**Review by sglanzer-fire — APPROVED** [review]

**Comment by sglanzer-fire** [comment]
Approved - thanks!

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-modal/pull-request/53/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-modal)

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:

## Changed

### Chrome: about-with-product

<table><tr> <td></td> <td></td> <td></td> </tr><tr> <td>Diff</td> <td>Current</td> <td>Baseline</td> </tr></table>

### Chrome: about-with-multiple-verions

<table><tr> <td></td> <td></td> <td></td> </tr><tr> <td>Diff</td> <td>Current</td> <td>Baseline</td> </tr></table>

### Chrome: about-with-product-and-multiple-verions

<table><tr> <td></td> <td></td> <td></td> </tr><tr> <td>Diff</td> <td>Current</td> <td>Baseline</td> </tr></table>

**Pull request merged**